src/modules/csm/csm.py

```diff
@@ -253,12 +256,15 @@ def collect_data(self, blockstamp: BlockStamp) -> bool:
         return self.state.is_fulfilled
 
     @lru_cache(maxsize=1)
-    def current_frame_range(self, blockstamp: BlockStamp) -> tuple[SlotNumber, SlotNumber]:
+    def current_frame_range(self, converter: Web3Converter, blockstamp: BlockStamp) -> tuple[SlotNumber, SlotNumber]:
```

Comment:
Why not to `converter = self.converter(blockstamp)`?

Reply:
to not to do it twice